vsintegration/src/FSharp.ProjectSystem.FSharp/ProjectFlagsService.fs

```diff
@@ -0,0 +1,123 @@
+﻿// Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.VisualStudio.FSharp.ProjectSystem
+
+open System
+open System.ComponentModel.Composition
+open System.Collections.Generic
+open Microsoft.VisualStudio.FSharp.LanguageService
+open Microsoft.VisualStudio
+open Microsoft.VisualStudio.Shell
+open Microsoft.VisualStudio.Shell.Interop
+open Microsoft.Build.Execution
+
+(*
+[<Export(typeof<ProjectFlagsService>)>]
```

Comment:
...then query it like this http://source.roslyn.io/#Microsoft.CodeAnalysis.EditorFeatures/CommandHandlers/GoToAdjacentMemberCommandHandler.cs,53